pkgs/development/python-modules/plotpy/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cython_0,
+  numpy_2,
+  setuptools,
+  guidata,
+  pillow,
+  qwt,
+  scikit-image,
+  scipy,
+  tifffile,
+}:
+
+buildPythonPackage rec {
+  pname = "plotpy";
+  version = "2.6.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "PlotPyStack";
+    repo = "PlotPy";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-/TV5JfnrG/lxtoUFoHjxL7+iw1/x1gVlfohS3tUipRk=";
+  };
+
+  build-system = [
+    cython_0
+    setuptools
+  ];
+
+  dependencies = [
+    guidata
+    # Doesn't support numpy_1
```

Comment:
You're sure?
https://github.com/PlotPyStack/PlotPy/blob/v2.6.2/pyproject.toml#L8-L9

Reply:
> You're sure? [PlotPyStack/PlotPy@`v2.6.2`/pyproject.toml#L8-L9](https://github.com/PlotPyStack/PlotPy/blob/v2.6.2/pyproject.toml?rgh-link-date=2024-09-01T16%3A28%3A34Z#L8-L9)

Haha thanks for the link - that's another argument for me to not propagate `numpy_1` nor `numpy_2` even for this package - to allow the user to choose!

Reply:
The user gets to choose either way. They can use `packageOverrides` if they want `numpy_2`.